trillian/ctfe/handlers.go

```diff
@@ -717,7 +722,7 @@ func getEntries(ctx context.Context, li *logInfo, w http.ResponseWriter, r *http
 	// The first job is to parse the params and make sure they're sensible. We just make
 	// sure the range is valid. We don't do an extra roundtrip to get the current tree
 	// size and prefer to let the backend handle this case
-	start, end, err := parseGetEntriesRange(r, MaxGetEntriesAllowed)
+	start, end, err := parseGetEntriesRange(r, MaxGetEntriesAllowed, strconv.FormatInt(li.logID, 10))
```

Comment:
Could you push this conversion down into `parseGetEntriesRange` (this is really an implementation detail of the counter label, and can be avoided if the feature is disabled).

Reply:
done